lib/bridge/IrcBridge.js

```diff
@@ -1048,6 +1050,19 @@ IrcBridge.prototype._fetchJoinedRooms = Promise.coroutine(function*() {
     }
 });
 
+
+IrcBridge.prototype._roomUpgradeMigrateEntry = function(entry, newRoomId) {
```

Comment:
nit: Might be nice to have a comment here along the lines of https://github.com/matrix-org/matrix-appservice-bridge/pull/129 for why this function is necessary.